gta/dff_importer.py

```diff
@@ -532,6 +537,8 @@ def import_frames():
             if index in self.meshes:
                 mesh = self.meshes[index]
 
+            if self.meshes_only and mesh is None: continue
```

Comment:
I added this, because in my case, I don't really need the collision empties, I only care about meshes. Would this be a possible option to add? (I hope this isn't causing the problems I had with importing...)

Reply:
Ok yeah, it caused some issues. My bad.
Still I'd like this option to be present. I'll try to implement it properly...

Reply:
I guess you could hide them instead.